src/main/java/seedu/address/logic/commands/iteration/AddIterationCommand.java

```diff
@@ -25,6 +30,7 @@ public class AddIterationCommand extends Command {
             + "Parameters: "
             + PREFIX_ITERATION_DATE + "DATE "
             + PREFIX_ITERATION_DESCRIPTION + "DESCRIPTION "
+            + PREFIX_ITERATION_IMAGEPATH + "IMAGE PATH "
             + PREFIX_ITERATION_FEEDBACK + "FEEDBACK\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_ITERATION_DATE + "2022-10-10 "
```

Comment:
Was going to comment on this on @nvmdava's PR but it closed- I think you forgot to update the `additer` example command with the example filepath?